src/test/java/ru/training/at/hw9/TrelloApiTests.java

```diff
@@ -0,0 +1,169 @@
+package ru.training.at.hw9;
+
+import static constants.Answers.INVALID_ID;
+import static constants.Answers.INVALID_KEY;
+import static core.TrelloServiceObject.badResponseSpecification;
+import static core.TrelloServiceObject.getBoardFromResponse;
+import static core.TrelloServiceObject.goodResponseSpecification;
+import static core.TrelloServiceObject.requestBuilder;
+import static core.TrelloServiceObject.unauthorizedSpecification;
+
+import beans.Board;
+import constants.ParameterName;
+import core.DataProviders;
+import core.TrelloServiceObject;
+import io.restassured.http.Method;
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+import properties.TrelloTestProperties;
+import ru.training.at.hw9.steps.BoardsSteps;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+
+public class TrelloApiTests {
+    TrelloTestProperties props = new TrelloTestProperties();
+    Board board;
```

Comment:
why do you need props - to read from property file?

Reply:
deleted the line with props

Reply:
+